kubelogs.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
```

Comment:
todo: clean this up significantly. 

While it was based on https://github.com/eduardobaitello/kubelogs/blob/master/kubelogs we can actually cut out almost everything.

Working with `kubectl` works sooo much better than the API in terms of stability, so I'm planning on sticking with using a bash script for `kubectl` access.

Reply:
I'll do this in the future though. No real reason to do it now.